### PR TITLE
GET specific template version

### LINKED ISSUE #2803: Unable to get specific version of a template
Using the API it's currently not possible to retrieve a specific template version. This is needed for resource updates, else the UI can only pull the latest template version - yet the resource is validated against the previous version. It can introduce scenarios where the resource is 'stuck' and unable to be updated.

### PR DESCRIPTION
**Pull request by damoodamoo**
Closes #2803 

Part of work needed to support #2795 

Makes it possible for clients to call the API to retrieve specific versions of the templates, rather than just the latest version.

### CODE REVIEW

**Review by tanya-borisova — APPROVED** [review]

File: api_app/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.5.9"
```
Comment (tanya-borisova) [inline api_app/_version.py:1]: Don't forget the changelog :)

Comment (damoodamoo) [reply api_app/_version.py:1]: gaaaaaaaaaaaaaaa David!

**Pull request merged**